### ISSUE TITLE
Bug Report: Playground no longer suggests collections with a `.` in the name when typing `db.`

### ISSUE THREAD

**Issue by benatshippabo** [mongodb-js/vscode#513]
#### Current Behavior

Currently, the MongoDB playground will not suggest all the collections when typing:

```
use('mydb')
db.
```

Some collections are suggested, but they disappear after typing a matching string. Example:

1. type db.
2. `animals.humans` is a suggested collection
3. continue typing `db.ani`
4. suggestion disappears

#### Environment

- node.js / npm versions: v18.16.0
- OS: MacOS 13.3.1 (22E261)

#### Possible Solution

Previously, mongo would suggest all the collections and pressing tab would complete the expression. Example:

1. type `db.ani`
2. `animals.humans` collection is suggested
3. pressing tab completes with `db['animals.humans']`

#### Additional Context/Screenshots

vscode info:
```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:41:48.638Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: No
```

extension version: `v0.11.1`

**Comment by alenakhineika** [comment]
Hey, sorry to hear you are seeing troubles with completions. Trying this myself with a standalone cluster and unfortunately can not reproduce it. Which version of the extension and VSCode are you using? Do you see all these collections on the sidebar? Is any of the collections being properly completed on tab?

**Comment by benatshippabo** [comment]
Hi @alenakhineika I just updated the description. I seem to be able to isolate this to collection names with a `.` in them. I tested this other collections names without a `.` and they work fine

**Comment by alenakhineika** [comment]
Could you try disconnecting and connecting to a cluster or click refresh for a current connection? I am wondering if you just created the collection and trying to use it right away. We currently do not update completions on the fly and a refresh is required to make them show up. I've just tried with the same collection name `animals.humans` and after refreshing this works. One thing that needs to be changed tho is to remove the extra `.` after `db` when converting it to `db['animals.humans']`.

**Comment by benatshippabo** [comment]
Hmm, it's still not working after reconnecting. I am also on MongoDB 5.0.14

**Comment by alenakhineika** [comment]
Would be very nice if you were able to share a screen recording to see exactly what is happening (make sure that there is no sensitive information on the screen if you do the recording).

**Comment by benatshippabo** [comment]
Okay here is the reproduction video:


https://user-images.githubusercontent.com/66273043/233209891-f211d884-c790-4bca-b840-2a2fbb894891.mov

**Comment by alenakhineika** [comment]
Could you try to use Disconnect/Connect instead of Refresh? (no need to record this again)

**Comment by benatshippabo** [comment]
Disconnect and connect also did not fix it

**Comment by alenakhineika** [comment]
Thank you very much for helping to catch this bug! Was able to reproduce and created a ticket: https://jira.mongodb.org/browse/VSCODE-407

**Comment by Anemy** [comment]
This is fixed in the 1.0.1 release - thanks again for opening the issue.
